Anlab.Mvc/Client/components/OrderForm.tsx

```diff
@@ -207,7 +207,7 @@ export default class OrderForm extends React.Component<undefined, IOrderState> {
                         <label className="form_header" id="quantity">How many samples will you require?</label>
                         <Quantity quantity={quantity} onQuantityChanged={this.onQuantityChanged} />
                     </div>
-                    <AdditionalEmails addedEmails={additionalEmails} onEmailAdded={this.onEmailAdded} onDeleteEmail={this.onDeleteEmail}/>
+                    <AdditionalEmails addedEmails={additionalEmails} onEmailAdded={this.onEmailAdded} onDeleteEmail={this.onDeleteEmail} defaultEmail={window.App.defaultEmail} />

```

Comment:
@laholstege can you set the defaultEmail to this.state for the order form component instead of using window here, so we can pin window.App access to just the ctor.